python/python/glide/async_commands/core.py

```diff
@@ -2877,6 +2879,90 @@ async def zdiffstore(self, destination: str, keys: List[str]) -> int:
             ),
         )
 
+    async def zinterstore(
+        self,
+        destination: str,
+        keys: Union[List[str], List[Tuple[str, float]]],
+        aggregation_type: Optional[AggregationType] = None,
+    ) -> int:
+        """
+        Computes the intersection of sorted sets given by the specified `keys` and stores the result in `destination`.
+        If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
+        When in cluster mode,`destination` and all keys in `keys` must map to the same hash slot.
```

Comment:
```suggestion
        If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.

        When in cluster mode, `destination` and all keys in `keys` must map to the same hash slot.
```